docs/schedule/README.md

```diff
@@ -54,14 +54,28 @@ You can create rooms by clicking on Admin on the top right, then Rooms.
 ### Options
 ![Reservation options](https://attachments.cbd.int/apps/eunomia/reservation-options.png)
 
-- In the Options tab, you can specify certain options specific to your event such as public accesseibility, confirmation status and security presence at the venue.
-- Similar to Details, information entered in the Options window will be shown on the Reservations page.
+- In the Options tab, you can specify certain options specific to your event such as public accessibility, confirmation status and security presence at the venue.
 
 ### CCTV
 ![Reservation cctv animated gif](https://attachments.cbd.int/apps/eunomia/reservation-cctv.gif)
 
-- Click on the CCTV tab to enter any text you would like to show on the CCTV screen
-- When you are done entering the event's details, click on Save to finalize the reservation
+- Click on the CCTV tab to enter any text you would like to be shown on the CCTV screen
```

Comment:
Maybe elaborate a little more.  Text with respect to a reservation shown on the cctv schedules.